gulpfile.js

```diff
@@ -0,0 +1,21 @@
+'use strict';
+
+var babel = require('gulp-babel');
+var metal = require('gulp-metal');
+var gulp = require('gulp');
+
+metal.registerTasks({
+	testDepTasks: ['build:cjs']
```

Comment:
This `build-cjs` task is only needed in the metal/metal.js repo because it has multiple packages, and the karma test configuration there uses their commonjs precompiled versions. Since this is using the default test config it won't need this task. You can actually just have this file register tasks without passing any options to gulp-metal, like this: `metal.registerTasks()`.